library/std/src/path.rs

```diff
@@ -1519,6 +1519,77 @@ impl PathBuf {
         true
     }
 
+    /// Append [`self.extension`] with `extension`.
+    ///
+    /// Returns `false` and does nothing if [`self.file_name`] is [`None`],
+    /// returns `true` and updates the extension otherwise.
+    ///
+    /// If [`self.extension`] is [`None`], the extension is added; otherwise
+    /// it is appended.
```

Comment:
Adding and appending mean the same thing. Is there a better way to explain the distinction being made here?

Reply:
I think it's no difference for `add_extension`. Let me remove this confusing comment.

It's derivated from the `set_extension` method where there is a difference.

Reply:
Updated.